gtk4/src/builder.rs

```diff
@@ -18,6 +18,20 @@ impl Builder {
         }
     }
 
+    #[doc(alias = "gtk_builder_get_current_object")]
+    #[doc(alias = "get_current_object")]
+    pub fn current_object(&self) -> Option<glib::Object> {
+        unsafe {
+            let ptr = ffi::gtk_builder_get_current_object(self.to_glib_none().0);
+            if ptr.is_null() {
+                None
+            } else {
+                let object: Borrowed<glib::Object> = from_glib_borrow(ptr);
+                Some(object.as_ref().clone())
```

Comment:
Stupid floating references :) How I worked around that in other places is via

```rust
glib::gobject_ffi::g_object_ref(ptr as *mut glib::gobject_ffi::GObject);
Some(from_glib_full(ptr))
```

But your version also works of course